controllers/configuration_controller.go

```diff
@@ -1007,6 +1015,16 @@ func (meta *TFConfigurationMeta) prepareTFVariables(configuration *v1beta2.Confi
 		valueFrom.SecretKeyRef.Name = meta.VariableSecretName
 		envs = append(envs, v1.EnvVar{Name: k, ValueFrom: valueFrom})
 	}
+	for k, v := range meta.ProviderVariable {
+		envValue, err := tfcfg.Interface2String(v)
+		if err != nil {
+			return err
+		}
+		data[k] = []byte(envValue)
+		valueFrom := &v1.EnvVarSource{SecretKeyRef: &v1.SecretKeySelector{Key: k}}
+		valueFrom.SecretKeyRef.Name = meta.VariableSecretName
```

Comment:
This implies provider credentials could be overridden?

Reply:
Yes, I think it's reasonable. Coverage behavior is the same as Region.